plugins/org.eclipse.n4js.ts.ui/src/org/eclipse/n4js/ts/ui/search/LabellingReferenceFinder.java

```diff
@@ -37,12 +39,16 @@ protected Acceptor toAcceptor(IAcceptor<IReferenceDescription> acceptor) {
 			@Override
 			public void accept(EObject source, URI sourceURI, EReference eReference, int index, EObject targetOrProxy,
 					URI targetURI) {
-				String name = labelProvider.getText(source);
-				LabelledReferenceDescription description = new LabelledReferenceDescription(sourceURI, targetURI,
-						eReference, index, name);
-				accept(description);
+				ICompositeNode srcNode = NodeModelUtils.getNode(source);
+				if (srcNode != null) {
+					// Only accept candidates with a corresponding node model.
+					// GH-73: TODO improve this
```

Comment:
probably no longer required, after making sure that TModule and "Ignored cached elements" are not traversed in ConcreteSyntaxAwareReferenceFinder